keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -366,17 +366,17 @@ kind: BucketAccessClass
 metadata: 
   name:
 provisioner: [1]
-policyActions: [2]
-  allow:
-  - "*"
-  deny:
-  - "*"
-parameters: [3]
+policyActionsConfigMap: [2]
```

Comment:
For simplicity, I think it's fine to just number this field (2) and not number 3 or 4 below. Either way for now.